Balance/Fealing/FeelingView.swift

```diff
@@ -31,14 +31,14 @@ struct FeelingView: View {
     var guessOption: some View {
         NavigationLink(
             destination: ActivityLogBaseView(
-                viewName: "Guess the Emoticon Feature",
+                viewName: "Guess the Emotion Feature",
                 isDirectChildToContainer: true,
                 content: {
                     GuessView()
                 }
             )
         ) {
-            DistractionCellView(image: "guessIcon", text: "Guess Emoticon", textDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s", pointVal: "+5")
+            DistractionCellView(image: "guessIcon", text: "Guess the emoticon", textDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s", pointVal: "+5")
```

Comment:
```suggestion
            DistractionCellView(image: "guessIcon", text: "Guess the emotion", textDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s", pointVal: "+5")
```